sc/src/main/java/uk/ac/ebi/atlas/experimentpage/ExperimentController.java

```diff
@@ -40,12 +40,19 @@ public String showExperimentPage(Model model,
         Experiment<?> experiment = experimentTrader.getExperiment(experimentAccession, accessKey);
         model.addAllAttributes(experiment.getAttributes());
 
+        if (!experiment.getDois().isEmpty()) {
+            model.addAttribute("publications", experimentPageContentService.getPublications(experiment.getDois()));
+        }
+        else if (!experiment.getPubMedIds().isEmpty()) {
+            model.addAttribute("publications", experimentPageContentService.getPublications(experiment.getPubMedIds()));
+        }
```

Comment:
Why isn’t the field `publication` populated as part of `experiment.getAttributes`?